addon/components/document-list-item.hbs

```diff
@@ -21,7 +21,7 @@
     <span>{{@document.title}}</span>
     <div
       class="document-thumbnail"
-      uk-dropdown="pos: right-bottom; offset: 15; delay-show: 400; delay-hide: 100;"
+      uk-dropdown="pos: right-center; offset: 15; delay-show: 400; delay-hide: 100;container: .alexandria-container;"
```

Comment:
Kind of angry at myself for leaving such a comment but my OCD can't help itself :joy: :
```suggestion
      uk-dropdown="pos: right-center; offset: 15; delay-show: 400; delay-hide: 100; container: .alexandria-container;"
```

Reply:
:sweat_smile: kind of angry at myself for not seeing it myself! fixed.